### ISSUE TITLE
A11y view doesn't work in published sims on phet-server

### ISSUE THREAD

**Issue by zepumph** [phetsims/chipper#828]
The code that loads the sim into the iframe in the a11y view is hard coded for the filename for phet-dev deploys, but the name changes in production mode. As a result this is broken here: https://phet.colorado.edu/sims/html/gravity-force-lab-basics/1.0.0/gravity-force-lab-basics_a11y_view.html.

Note that each time a sim is published to the website, a copy is also published to phet-dev, such that even though the above doesn't work, we can still access the exact same file here: https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0/phet/gravity-force-lab-basics_a11y_view.html

Also tagging @jessegreenberg as the original author. In terms of fixing, I'm a bit nervous about complexity. In PhET-iO we went through a few iterations of logic to detect what sim to load from the "wrapper" (the page that has the sim in an iframe). Still though it may be as easy as trying to get one file, and if it fails then loading a different one.

@emily-phet please prioritize this issue. Does it feel like something we should fix? I think this falls more on the branding side than anything else. Is it problematic to show outside individuals links on phet-dev vs having the link come from phet.colorado.edu.

**Comment by emily-phet** [comment]
As long as we can access the a11y view for the published sim, regardless of filename, I think we're fine as is. If you see a use-case where the current situations is problematic (beyond just the fact that a11y-view users need to know to use the phet-dev filename) let me know.

**Comment by zepumph** [comment]
I'm good closing this, @jessegreengerg please do so if you agree.

**Comment by jessegreenberg** [comment]
Thanks for pointing out. I am fine with a11y-views going through phet-dev. Should we exclude a11y-view from production build task? Not that it takes up much space but may be best to not put a broken file up there.

Looks like the a11y-view is generated buildRunnable.js, I am not sure if there is awareness of "production" at that phase. @zepumph are you familiar?

**Comment by zepumph** [comment]
We are trying to remove the a11y view build artifact so that it doesn't end up on the production server, but it does end up on the same build's copy on phet-dev. To me it may be more of a headache than it is worth to try to split those up. It would also be a build-server change as opposed to just chipper (which I believe is meant to build a destination-independent bundle).

**Comment by jessegreenberg** [comment]
Got it, OK thanks. Alright, closing this issue.
